server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -828,6 +832,60 @@ const orderMutations = {
         },
       });
 
+      return order;
+    },
+  },
+  editPendingOrder: {
+    type: new GraphQLNonNull(Order),
+    description: 'To submit a new order. Scope: "orders".',
+    args: {
+      order: {
+        type: new GraphQLNonNull(PendingOrderEditInput),
+      },
+    },
+    async resolve(_, args, req) {
+      if (!checkScope(req, 'orders')) {
+        throw new Unauthorized('The User Token is not allowed for operations in scope "orders".');
+      }
+
+      const order = await fetchOrderWithReference(args.order, {
+        throwIfMissing: true,
+        include: [{ model: models.Collective, as: 'collective' }],
+      });
+
+      const host = await order.collective.getHostCollective();
+      if (!req.remoteUser?.isAdminOfCollective(host)) {
+        throw new Unauthorized('Only host admins can process orders');
+      }
+
+      if (!(await canEdit(req, order))) {
+        throw new ValidationFailed(`Only pending orders can be edited, this one is ${order.status}`);
+      }
+
+      const fromAccount = await fetchAccountWithReference(args.order.fromAccount);
+
+      await order.update({
+        CreatedByUserId: req.remoteUser.id,
+        FromCollectiveId: fromAccount?.id || undefined,
+        totalAmount: args.order.amount.valueInCents,
```

Comment:
We need to keep supporting passing the amount via `amount.value` since the API exposes it

```suggestion
        totalAmount: getValueInCentsFromAmountInput(args.order.amount),
```